nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -435,8 +440,12 @@ def create_accuracy_aware_training_loop(nncf_config: NNCFConfig,
     """
     accuracy_aware_training_params = extract_accuracy_aware_training_params(nncf_config)
     accuracy_aware_training_mode = accuracy_aware_training_params.get('mode')
+    eval_fn = nncf_config.get_extra_struct(ModelEvaluationArgs).eval_fn
```

Comment:
This will require some changes in OTX integration. For example, OTX provides `nncf_config` of instance `mmcv.ConfigDict`, not `NNCFConfig` as it should be.

cc @cih9088